src/content/post/20160405_bbh.md

```diff
@@ -0,0 +1,51 @@
+
+In the first BBH of 2016 spring Marcello Testi discussed about Drush Make related issues within v8 of Drush, and Paolo Mainardi introduced the Hugo static site generator to the team in order to kickstart the [SparkFabrik] Tech Blog. We also talked a little about design thanks to our weekly news collection.
+
+<!--more-->
+
+# Weekly News
+
+This week we discussed mainly design-related news, brought by Marcello Gorla.
+
+### Adobe XD
+
+![Adobe XD](/posts/2016_adobexd.jpg)
+
+Originally know as “Project Comet", Adobe has released the preview of [Adobe XD](http://blogs.adobe.com/creativecloud/introducing-adobe-experience-design-cc-preview/), it's own tool oriented to design and prototype user experience projects. The tool it's an early stage release that still does not offer a full stack of features as we are used to with more advanced (and mature) softwares like Sketch or Adobe Photoshop, but there's a [lot of stuff moving on](https://www.youtube.com/playlist?list=PLD8AMy73ZVxXvBQcZAnOcu57JCFxLi7bQ) and we think it's worth trying.
+
+[Some opinions](https://medium.com/user-experience-design-1/first-impressions-with-adobe-experience-design-107904b5efbe#.25cvqr3ys) of course have been [already popped out](http://techcrunch.com/2016/03/14/adobe-launches-experience-design-cc-a-new-tool-for-ux-designers/).
+
+### Frog will have no more CCO
+
+[Frog](http://www.frogdesign.com/), the international design firm, after the departure of Hans Neubert has recently decided [not to replace it's Chief Creative Officer with a new one](http://designmind.frogdesign.com/2016/03/evolving-creative-leadership-at-frog/), prefering a more agile structure with many Creative Officers around the world leading the single design teams. This event has an interesting impact if seen from a wider point of view.  A [Fast Co Design] article made a good point over this, trying to consider what is happening to the design consultancies and firms and to the field in general.
+
+> While design is more in demand than ever, the business of design consultancies has been in flux. Mega business consultants Accenture and IBM have each been making $100 million-plus bets on increasing their design fluency. Meanwhile, design firms ranging from Lunar to Ideo have reassessed their business strategies, and firms like Yves Behar's Fuseproject have sold outright to foreign investors.
+(Via [Fast Co Design])
+
+### OLO reached founding goal
+
+![Adobe XD](/posts/2016_olo.jpg)
+
+The italian kickstarter project [Olo](https://www.kickstarter.com/projects/olo3d/olo-the-first-ever-smartphone-3d-printer), a 3D printer suited for the smartphones screen, has reached the funding goal and is running fast towards more stellar ones. Being very low-cost but promising a very good print resolution, this 3D printer could be a step forward to unchain the potential of personal projects prototyping and experimentation, but also a new frontier to many other unexplored fields like the home-productions of games.
```

Comment:
`italian` -> `Italian`. I recently discovered that, unlinke Italian, nationality adjectives in English are always capitalized.